pandas/core/base.py

```diff
@@ -152,6 +153,9 @@ class SelectionMixin:
     object sub-classes need to define: obj, exclusions
     """
 
+    obj: Any
+    exclusions: Set[Label] = set()
```

Comment:
looks like the docbuild failure is bc this is public but undocumented

Reply:
The docstring states `exclusions : set, optional`. Is the docstring referring to required class attributes? The docstring states `sub-classes need to define: obj, exclusions`. why is `exclusions` initialised here?

Reply:
``exclusions`` is initialized here to deal with the following mypy errors.

```
pandas\core\base.py:230: error: "SelectionMixin" has no attribute "exclusions"  [attr-defined]
pandas\core\base.py:231: error: "SelectionMixin" has no attribute "exclusions"  [attr-defined]
```

Reply:
no need to initialize if sub-classes need to define as stated in the docstring, declaration will silence those errors.

```suggestion
    exclusions: Set[Label]
```